openedx/core/djangoapps/xblock/rest_api/views.py

```diff
@@ -36,8 +36,7 @@
 }
 
 
-class InvalidNotFound(NotFound):
-    default_detail = "Invalid XBlock key"
+InvalidNotFound = "XBlock {usage_key} does not exist, or you don't have permission to view it."
```

Comment:
nitpick: More Pythonic variable name.

```suggestion
invalid_not_found_fmt = "XBlock {usage_key} does not exist, or you don't have permission to view it."
```